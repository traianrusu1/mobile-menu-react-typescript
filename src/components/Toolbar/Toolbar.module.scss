@import "../../appStyles/variables";
@import "../MobileMenu/mobileMenuVariables";


.toolbar {
  background-color: $appColor1;
  color: $fontColorWhite;
  height: $toolbarHeight;
  z-index: 200;
  box-sizing: border-box;

  button {
    background-color: transparent;
    margin-top: 3px;
    margin-left: 3px;
    padding: 10px 15px;
    border: none;
    // margin-bottom: 10px;
    cursor: pointer;
    color: $fontColorWhite;
  }
}
