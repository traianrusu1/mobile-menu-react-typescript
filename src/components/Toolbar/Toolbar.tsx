import React from 'react';
import styles from './Toolbar.module.css';

interface Props {
  myProp: string;
}

const Toolbar: React.FC<Props> = ({ myProp }: Props) => {
  return (
    <div className={styles.toolbar}>
      Toolbar
      {myProp}
    </div>
  );
};

export default Toolbar;
