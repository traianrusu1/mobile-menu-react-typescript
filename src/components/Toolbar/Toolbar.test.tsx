import React from 'react';
import { render, cleanup } from '@testing-library/react';

import Toolbar from './Toolbar';

afterEach(cleanup);

describe('<Toolbar />', () => {
  test('renders', () => {
    const { container } = render(<Toolbar myProp="" />);
    expect(container).toMatchSnapshot();
  });
});

