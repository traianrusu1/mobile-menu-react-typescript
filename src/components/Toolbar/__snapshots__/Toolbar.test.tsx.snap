// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toolbar /> renders 1`] = `
<div>
  <div
    class="toolbar"
  >
    Toolbar
    
  </div>
</div>
`;
